### DISCUSSION TITLE
Add our MLH guild EddieHub to our community website

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#662] [First-Timers]
Make it easier for people to join our guild, with links and steps

http://eddiejaoudecommunity.github.io

**Comment by blackpandan** [comment]
I am ready for the task

**Comment by mikeysan** [comment]
Sounds like a great idea

**Comment by eddiejaoude** [comment]
oops I think we all got busy with the hackathon 😂 

We could still add some info about it and links to the hashtag etc

**Reply by Vyvy-vi** [reply]
seems late, maybe we could add a section to the website, for `Our Hackathons`, and show hacktoberfest and MLH LHD: Build data there?

**Reply by BOLT04** [reply]
Great idea, I'm in favor of creating an issue for that @Vyvy-vi 👍 👍

**Reply by Akshu-on-github** [reply]
I agree! And LHD: Share is a few months from now, if we participate in it and other such Hackathons, it would be easy organisation

**Reply by eddiejaoude** [reply]
Awesome 👍
